app-hw-37/src/Main/Main.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactNode } from 'react';
+import { useState, useEffect } from 'react';
+import { CardsList } from '../CardsList/CardsList';
+import { FullScreenCard } from '../FullScreenCard/FullScreenCard';
+
+type Post = {
+	id: number,
+	image?: string,
+	text: string,
+	date: string,
+	lesson_num: number,
+	title: string,
+	description: string,
+	author: number
+}
+
+
+export const Main = () => {
+	const [page, setPage] = useState('List')
+	const [fullScreenPostId, setFullScreenPostId] = useState(0)
```

Comment:
prefer null or undefined then 0

Reply:
gives TS mistake: Argument of type 'number' is not assignable to parameter of type